spec/operators/zip-spec.ts

```diff
@@ -595,4 +596,34 @@ describe('Observable.prototype.zip', () => {
     expectSubscriptions(a.subscriptions).toBe(asubs);
     expectSubscriptions(b.subscriptions).toBe(bsubs);
   });
+
+  it('should support rest parameter observables', () => {
+    type(() => {
```

Comment:
`type` is actually have signature of `type(title: string, test: fn)` so doesn't require wrap via `it`. I see current codebases have this practices, which need to be updated as well.